docker/py3-native/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM demisto/py3-tools-ubi:1.0.0.98074
+FROM devdemisto/py3-tools-ubi:1.0.0.92372
```

Comment:
devdemisto

Reply:
Can't update until the [py3-tools-ubi](https://github.com/demisto/dockerfiles/pull/25902) PR is merged